src/internal/operations/backup_integration_test.go

```diff
@@ -455,6 +455,29 @@ func toDataLayerPath(
 	return p
 }
 
+func mustGetDefaultDriveID(
+	t *testing.T,
+	ctx context.Context, //revive:disable-line:context-as-argument
+	service graph.Servicer,
+	userID string,
+) string {
+	d, err := service.Client().UsersById(userID).Drive().Get(ctx, nil)
+	if err != nil {
+		err = graph.Wrap(
+			ctx,
+			err,
+			"retrieving default user drive").
+			With("user", userID)
+	}
+
+	require.NoError(t, err)
```

Comment:
Nit: add the clues error core for debugging
```suggestion
	require.NoError(t, err, clues.ToCore(err))
```